### DISCUSSION TITLE
149

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#750] [Comments]
http://jojoldu.tistory.com/149

**Comment by jojoldu** [comment]
작성자: 검둥개

깔끔하게 잘 정리하셨네요. 감사합니다.

**Comment by jojoldu** [comment]
작성자: 창천향로 창천향로

감사합니다^^

**Comment by jojoldu** [comment]
작성자: 하비

좋은 글 감사합니다! 모든 버튼을 다 몰랐었는데 ㅎㅎ
알고 계시겠지만 글에 해당 내용이 포함되지 않은것 같아서...Evaluate의 Code Fragment Mode 사용시 
한 줄(메소드 변수, 클래스 필드)뿐만 아니라 여러 줄에 걸쳐 테스트가 가능해 코딩 혹은 디버깅하는데 시간절약이 되더라구요.
여러 줄에 걸친 코드를 테스트 하고 싶은 경우 참고하시기 바랍니다ㅎㅎ;;

**Comment by jojoldu** [comment]
작성자: 블랙환

굿굿~~~!!
인텔리J 신~~~!!!!
책이나 유료강의하시면 꼭 듣고 싶어요

**Comment by donghyeon0725** [comment]
이런 좋은 글을 작성해주셔서 정말 감사합니다..
                                  -- 지나가던 초보 개발자가

**Comment by brucehaan** [comment]
인텔리제이로 이제 막 디버깅 접했는데 선생님 포스팅 덕분에 쉽게 접근할 수 있게 되었습니다. 감사합니다🥰

**Comment by SUPER0214T** [comment]
좋은 글 감사합니다! Watch, Run to Cursor 정말 유용하게 사용할 것 같습니다!
